networking/network_security/AdminNetworkPolicy/ovn-k-anp-recommended-practices.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ovn-k-anp-recommended-practices"]
+= Best practices for AdminNetworkPolicy
```

Comment:
We also talk about BANP, would this title still sound correct? I don't have enough context so just confirming.

Reply:
ANP refers to both `AdminNetworkPolicy` the API object and `BaselineAdminNetworkPolicy` API object. So ANP discussed in a general sense refers to both APIs.